<launch>
<node name="network_delay_model" pkg="network_delay_model" type="network_delay_model_node" output="screen">
      <param name="virtual_object_position_sub_topic" value="/uav1/mavros/local_position/pose"/>
      <param name="virtual_object_position_pub_topic" value="/virtual/uav0/current_position"/>
      <param name="physical_object_position_sub_topic" value="/physical_uav/position"/>
      <param name="physical_object_position_pub_topic" value="/physical_uav/position/network"/>
      <param name="reset_object_sub_topic" value="/V_UAV_2/reset_object"/>
      <param name="predict_window" value="8"/>                                                     <!--The window size for predicting further system state(predicted steps)-->
      <param name="predict_time_dt" value="0.33"/>                                                  <!--The predicted time step for least square-->
      <param name="target_position_pub_topic" value="/controller/target/position"/>
      <param name="virtual2physical_delay_time_sub_topic" value="/delay_time/v2p"/>
      <param name="physical2virtual_delay_time_sub_topic" value="/delay_time/p2v"/>
      <param name="v2p_noise_mu" value="0.0"/>
      <param name="v2p_noise_sigma" value="5.001"/>
      <param name="p2v_noise_mu" value="0.0"/>
      <param name="p2v_noise_sigma" value="5.001"/>
      <param name="virtual_guide_type_sub_topic" value="/virtual_guide/type"/>
      <param name="virtual_guide_type_pub_topic" value="/network/virtual_guide/type"/>
      <param name="virtual_guide_cmd_sub_topic" value="/virtual_guide/cmd"/>
      <param name="virtual_guide_cmd_pub_topic" value="/network/virtual_guide/cmd"/>
      <param name="physical_uav_position_sub_topic" value="/physical_uav/position"/>
      <param name="physical_uav_position_pub_topic" value="/network/physical_uav/position"/>
      <param name="physical_uav_move_state_pub_topic" value="/network/physical_uav/move/state"/>

      </node>
</launch>
